demo/src/screens/componentScreens/ActionBarScreen.tsx

```diff
@@ -19,12 +19,12 @@ export default class ActionBarScreen extends Component {
 
   render() {
     return (
-      <View flex bg-grey80>
```

Comment:
Something's odd, I don't see the screen as dark, is it ok for you (merge master)?